src/common/index.ts

```diff
@@ -0,0 +1,26 @@
+import apiMod from './api'
```

Comment:
silly naming here cause eslint doesn't support the syntax
```ts
export * as api from './api'
```
& i couldn't just disable it